### ISSUE TITLE
Controlled generation with Gemini 1.5 Flash fails

### ISSUE THREAD

**Issue by fatihyildizhan** [GoogleCloudPlatform/dotnet-docs-samples#2770]
Hello,

I am using Google.Cloud.AIPlatform.V1 3.6.0 with .NET 8. Trying to run `GenerateContentWithResponseSchema6` but I get `Grpc.Core.RpcException: 'Status(StatusCode="InvalidArgument", Detail="Request with response_schema config is not supported.")'` exception.

This is the tutorial (generativeaionvertexai_gemini_controlled_generation_response_schema_6)
https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/main/aiplatform/api/AIPlatform.Samples/ControlledGeneration.cs#L361

Thank you

**Comment by meteatamel** [comment]
Hi @fatihyildizhan, I just ran the `TestGenerateContentWithResponseSchema6` test in `ControlledGenerationTest` which exercises the code you mentioned multiple times but it worked for me every single time. I wonder what might be different in your case?

**Comment by fatihyildizhan** [comment]
@meteatamel Thank you very much. Yes, I was trying it with Flash. Sorry for keeping you busy.
It was great that you shared this news, thank you for that.

**Comment by fatihyildizhan** [comment]
Hi @meteatamel 

Controlled generation is available for Flash and this is amazing.

I think my question is not related to dotnet-docs-samples but, I want to ask if you are experiencing the same situation.

Sometimes it can take up to 50 seconds for the response to return with "Controlled Generation". Are you experiencing this situation?

Thank you very much for your time.

![5 seconds](https://github.com/user-attachments/assets/df973fb0-3a58-43e6-9417-a4290bdc6f7d)
![50 seconds](https://github.com/user-attachments/assets/81454121-f5bd-4f49-8f93-9c8ab21d8243)

**Comment by meteatamel** [comment]
No, I didn't experience that but if you give me a runnable `Program.cs` file, I can try and see if it's a problem for me too.
